.github/workflows/update-test.yaml

```
@@ -79,6 +79,9 @@ jobs:
         PG_MAJOR=$(echo "${{ matrix.pg }}" | sed -e 's![.].*!!')
         UPDATE_FROM_TAG=${DOWNGRADE_TO}-pg${PG_MAJOR}
         export UPDATE_FROM_TAG
+        # Use specific version of alpine so that openssl version wont change
+        PG_IMAGE_TAG="${PG_VERSION}-alpine3.16"
+        export PG_IMAGE_TAG
         scripts/test_downgrade_from_tag.sh
```

```suggestion
        PG_IMAGE_TAG="${PG_VERSION}-alpine3.16" scripts/test_downgrade_from_tag.sh
```
Nit. It is up to you!!!